### PR TITLE
Benchmark CSV File Output

### PR DESCRIPTION
**Pull request by Cheryl-Lao**
Integrated thread logging into the existing benchmarks. Changed the output location to a csv file. Added variance and individual run info to the results

### CODE REVIEW

**Comment by gzinck** [comment]
This looks good!

Git repo bonus points: I would consider adding some of the output files to the `.gitignore` so that we don't keep pushing those to the repo every time somebody tries to run things. Recommended items to ignore:
- `benchmarks/PC_status_log.txt`
- `benchmarks/__pycache__/`
- `benchmarks/results.csv`

Also, it might be worthwhile to output the results to a separate folder, `/output`, instead. Not a big deal, but the output folder in the container is mounted to the repo's `output/` folder.

**Comment by Cheryl-Lao** [comment]
@gzinck Thanks for the review! I just updated it to move the outputs and ignore the pycache folder

File: benchmarks/PC_status_log.txt [hunk 1]
```diff
@@ -0,0 +1,197 @@
+7.8,0.0,0.08333333333333334
```
Comment (gzinck) [inline benchmarks/PC_status_log.txt]: Can delete this file now :)

File: benchmarks/results.csv [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Benchmark,Image Path,Binary Path,Kernel Width,Average Time Elapsed (seconds),Variance of Time Elapsed,Time Elapsed 1,Time Elapsed 2,Time Elapsed 3,Time Elapsed 4,Time Elapsed 5,Time Elapsed 6,Time Elapsed 7,Time Elapsed 8,Time Elapsed 9,Time Elapsed 10
```
Comment (gzinck) [inline benchmarks/results.csv]: Can also delete this one!

**Review by gzinck — COMMENTED** [review]
Heck yeah! Looks good!

**Review by gzinck — APPROVED** [review]
🎉🎉🎉

**Pull request merged**